### ISSUE TITLE
Downloaded but can't use

### ISSUE THREAD

**Issue by ComputerErika** [Joexv/OWOer#2]
So I downloaded this iMessage Extension. I went to try it came up with this, when I try to click [Tap here to get started] it wouldn't work. At the bottom, it has a cut-off sentence of "In order to use the iMessage extension you must purchase the ad..". I'm not sure what it wants me to purchase like maybe the addon it's self? Yet when I got it didn't charge me, it just said "In-app purchases". Help would be appreciated :)

![image](https://user-images.githubusercontent.com/42761313/68523459-5c837a80-026e-11ea-82f3-3f7001fcb4c8.png)

**Comment by Joexv** [comment]
Can you send me a screen shot of the settings/in app purchase page of the main app?

-Joe

**Comment by ComputerErika** [comment]
It's an app... When I downloaded it on the iMessage Appstore I thought it was just an iMessage extension... Wow, I was dumb. Well, thank you for showing me it was an app as well. Which fixes the issue of not being able to use it. And now I get what it means to purchase no adds for you to use the iMessage extension. So I guess it's fixed now but here's a screenshot of the setting anyways. The option sliders are mostly off the screen but they still worked when I tried them. Anyway, thank you for responding so quickly and helping me! :)

![image](https://user-images.githubusercontent.com/42761313/68524675-6d3bec80-027e-11ea-9fee-5f44900b587d.png)
